modules/docs/src/Volo.Docs.Admin.Application.Contracts/Volo/Docs/Admin/Localization/Resources/Docs/ApplicationContracts/ar.json

```diff
@@ -37,11 +37,11 @@
     "DisplayName:LanguageCode": "رمز اللغة",
     "DisplayName:Version": "إصدار",
     "Documents": "وثائق",
-    "RemoveFromCache": "حذف من الذاكرة",
+    "RemoveFromCache": "إزالة من ذاكرة التخزين المؤقت وإعادة الفهرس",
     "Reindex": "إعادة الفهرسة",
     "ReindexCompleted": "تمت إعادة الفهرسة",
-    "RemovedFromCache": "تم الحذف من ذاكرة التخزين",
-    "RemoveFromCacheConfirmation": "هل أنت متأكد من حذف هذا العنصر من ذاكرة التخزين؟",
+    "RemovedFromCache": "تمت إزالته من ذاكرة التخزين المؤقت وإعادة فهرسته",
+    "RemoveFromCacheConfirmation": "هل تريد بالتأكيد إعادة فهرسة  \"{0} \"؟",
     "ReIndexDocumentConfirmation": "هل أنت متأكد من إعادة فهرسة هذا العنصر؟",
```

Comment:
Please do the following changes:

* Revert the localization changes for "RemoveFromCache", "RemovedFromCache" and "RemoveFromCacheConfirmation".
* In addition to that, ensure all of the "ReIndexDocumentConfirmation" localization keys have been updated and take a parameter. (For example, it's not done for ar.json)